Automation/create_Azure_StorageAccount.ps1

```diff
@@ -0,0 +1,44 @@
+Start-Transcript -Path "$PSScriptRoot\create_Azure_StorageAccountlog.log"
+Write-Host "Logging to $PSScriptRoot\create_Azure_StorageAccountlog.log"
+Write-Host "Create Azure Storage Account"
+
+# Name of resource group
+$resourceGroup = "nsc-ad440-winter2021-Th-StorageRG"
```

Comment:
I was a little hesitant to do what you suggested in slack where I take multiple inputs and combine them into the RG name since that would just lead to a bunch of similarly named but different RGs. Would prompting for the RG name be sufficient?